eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -85,7 +85,7 @@ jobs:
             EnvVars:
               Pool: $(Pool)
               ${{ insert }}: ${{ parameters.EnvVars }}
-            ${{ if parameters.UseFederatedAuth }}:
+            ${{ if eq('true', parameters.UseFederatedAuth) }}:
```

Comment:
Are you doing this for consistency or does this one also require it?

Reply:
This works using the `${{ if parameters.UseFederatedAuth }}` syntax (eq expression not needed)